extensions/sql-database-projects/src/test/projectController.test.ts

```diff
@@ -429,128 +414,70 @@ describe('ProjectsController', function (): void {
 			should(spy.calledWith(msg)).be.true(`showErrorMessage not called with expected message '${msg}' Actual '${spy.getCall(0).args[0]}'`);
 		});
 
-		it('Should show error when no project name provided', async function (): Promise<void> {
```

Comment:
this test changed to 'ok" button not enabled kind of test - is my understanding correct?

Reply:
That is correct. These tests have been moved to createDatabaseFromDatabaseDialog.test.ts